gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/transports/rest.py.j2

```diff
@@ -66,7 +66,7 @@ class {{ service.name }}RestInterceptor:
     These use cases and more can be enabled by injecting an
     instance of a custom subclass when constructing the {{ service.name }}RestTransport.
 
-    .. code-block:
+    .. code-block: python
```

Comment:
```suggestion
    .. code-block:: python
```

Reply:
I don't see this being reflected in the Goldens below, so I'm not sure whether this should come with 2 colon characters or just one...